docker-compose.yml

```diff
@@ -20,22 +20,22 @@ services:
       - all
       - localstack
     environment:
-      DATA_DIR: /tmp/localstack/data
       # `kinesalite` provides a more accurate implementation than
       # the default Kinesis provider (`kinesis-mock`).
       KINESIS_PROVIDER: kinesalite
       SERVICES: kinesis,s3
+      PERSISTENCE: 1
     volumes:
-      - ".localstack:/etc/localstack/init/ready.d"
-      - "${TMPDIR:-/tmp}/quickwit/services/localstack:/var/lib/localstack"
+      - .localstack:/etc/localstack/init/ready.d
+      - localstack_data:/var/lib/localstack
     healthcheck:
       test: ["CMD", "curl", "-k", "-f", "https://localhost:4566/quickwit-integration-tests"]
       interval: 1s
       timeout: 5s
       retries: 100
 
   postgres:
-    image: postgres:9.6.20 # Oldest version we support.
+    image: postgres:${POSTGRES_VERSION:-9.6.20} # Oldest version we support.
```

Comment:
9 and 10 are no longer supported. Shall we set `11.19`?

Reply:
Sure, I will fix that. I thought it was about the oldest version **we** support, not the version that a vendor supports.